src/tools/cephfs_mirror/PeerReplayer.cc

```diff
@@ -120,7 +120,6 @@ int opendirat(MountRef mnt, int dirfd, const std::string &relpath, int flags,
 
   int fd = r;
   r = ceph_fdopendir(mnt, fd, dirp);
-  ceph_close(mnt, fd);
```

Comment:
I think ceph_close is necessary here. We are getting the dirp using `ceph_fdopendir`. Any thoughts @vshankar ?
https://github.com/ceph/ceph/blob/main/src/client/Client.cc#L9175

Reply:
No, it's not. Please see https://github.com/ceph/ceph/pull/55619 which states:
"Based on posix specification, the fd passed to fdopendir() will be closed by closedir(). However CephFS client wasn't doing that. If the user opened a directory using ceph_openat(), for example, and then passed the returned fd to ceph_fdopendir(), the created Fh associated with the new open was never destroyed.
"